Tone/event/Sequence.ts

```diff
@@ -30,7 +30,7 @@ interface SequenceOptions<T> extends Omit<ToneEventOptions<T>, "value"> {
  * Tone.Transport.start();
  * @category Event
  */
-export class Sequence<ValueType = any> extends ToneEvent<ValueType> {
+export class Sequence<ValueType extends Time | null = Time | null> extends ToneEvent<ValueType> {
```

Comment:
I'm not following the use of `Time` type here, if the sequence is like in some of the examples using Notes (e.g. `new Sequence((time, note) => {}, ["C4", "D4", "E4"])`), then the type should be `unknown | null` right?

Reply:
Ah yeah I was just looking at the tests and not considering the full context there – bounds have been removed!